### DISCUSSION TITLE
[守卫中canActivate返回false时，code总是500而不是403，且无法改变]

### DISCUSSION THREAD

**Discussion by ruiruiStudy** [midwayjs/midway#3920] [Q&A]
### Describe the problem（描述问题）

文档描述：默认情况下，守卫的 canActivate 方法当返回 false 时，框架会抛出 403 错误（ForbiddenError）。
实际上，框架抛出500错误，且无法改变值

### Midway Versions（Midway 版本）

1.3.8

**Comment by czy88840616** [comment]
检查自定义全局过滤器。
